src/locales/fr/home.json

```diff
@@ -0,0 +1,12 @@
+{
+  "welcome": {
+    "title": "Bienvenue sur 🚀 Start UI [native]",
+    "description": "Un starter UI opinioné avec Expo, Ficus UI, Zodios & Formiz",
```

Comment:
```suggestion
    "description": "Un starter UI dogmatique avec Expo, Ficus UI, Zodios & Formiz",
```

Reply:
Here we have Zodios but not React Query, and in onboarding we have React Query and not Zodios.
Please put only Zodios everywhere 😄